src/Modals/ModalBody.tsx

```diff
@@ -10,6 +10,7 @@ interface ModalBodyProps {
 
 const StyledModalBody = styled.div<ModalBodyProps>`
   padding: ${({ theme }): string => theme.ws.medium};
+  overflow: scroll;
```

Comment:
`overflow: auto` might be better to hide the scrollbar unless needed

Reply:
Interesting, does the scrollbar always show up for you? It only shows up for me when there's an overflow!

Reply:
I updated the overflow property to `auto` through commit 2437447.

Reply:
Yeah, I suspect that's MacOS being fancy :)

Generally it shows up all the time if you use `overflow: scroll`, whereas `overflow: auto` has the behavior you're describing.

https://stackoverflow.com/questions/6689412/difference-between-html-overflow-auto-and-overflow-scroll#:~:text=Auto%20will%20only%20show%20a,and%20you%20cant%20scroll%20it.&text=overflow%3A%20scroll%20will%20hide%20all,on%20the%20element%20in%20question.

Reply:
Ah good to know thanks for sharing that!